README.rst

```diff
@@ -16,34 +16,49 @@ Library to clean up Ansible tasks from any Personally Identifiable Information (
 * Free software: Apache Software License 2.0
 
 
-Features
---------
+Usage
+-----
 
 The library can be used to remove the PII from a multi level structure:
 
-.. code-block::
+.. code-block:: python
 
-   $ python3
-   >>> from ansible_anonymizer import anonymizer
-   >>> example = [{"name": "foo bar", "email": "my-email@address.com"}]
-   >>> anonymizer.anonymize_struct(example)
-   ['- email: lucas27@example.com\n  name: foo bar\n']
+   from ansible_anonymizer import anonymizer
+   example = [{"name": "foo bar", "email": "my-email@address.com"}]
+   anonymizer.anonymize_struct(example)
+   # ['- email: lucas27@example.com\n  name: foo bar\n']
```

Comment:
Though this line was just commented out and not updated, is it the expected output? I thought `.anonymize_struct` kept the structure of the input as it was.

Reply:
Owa, thanks! Let's me fix that.